nethermind/Dockerfile

```
@@ -0,0 +1,49 @@
+FROM golang:1.21 AS op
+
+WORKDIR /app
+
+ENV REPO=https://github.com/ethereum-optimism/optimism.git
+ENV VERSION=v1.9.1
+ENV COMMIT=4797ddb70e05d4952685bad53e608cb5606284e6
+RUN git clone $REPO --branch op-node/$VERSION --single-branch . && \
+    git switch -c branch-$VERSION && \
+    bash -c '[ "$(git rev-parse HEAD)" = "$COMMIT" ]'
+
+RUN cd op-node && \
+    make VERSION=$VERSION op-node
+
+FROM mcr.microsoft.com/dotnet/sdk:8.0-noble AS build
+
+ARG BUILD_CONFIG=release
+ARG TARGETARCH
+
+WORKDIR /app
+
+ENV REPO=https://github.com/NethermindEth/nethermind.git
+ENV VERSION=1.29.0
+ENV COMMIT=917f5349517118bf5c73c362c3902d61e68e5b40
+RUN git clone $REPO --branch release/$VERSION --single-branch . && \
```

Could we depend on a tagged commit rather than a branch (perhaps when the `1.29.0` release is tagged)? Otherwise new commits on that branch will break the build.

---

Yes, shipping this week likely

Adding tweaks from testing Base holding it up; want to make sure the release aligns well 😉

---

```suggestion
RUN git clone $REPO --branch $VERSION --single-branch . && \
```
to switch to tag rather than branch